kg_covid_19/transform_utils/scibite_cord/scibite_cord.py

```diff
@@ -175,6 +175,18 @@ def parse_annotation_doc(self, node_handle, edge_handle, doc: Dict) -> None:
                 curie = self.contract_uri(t)
                 name = self.concept_name_map[t] if t in self.concept_name_map else "",
 
+###
+###temporary solution to normalize PR: prefixes to UniProtKB
+###
+
+            ###test
+            if not re.search("^PR:\d+$", curie):
+                continue
```

Comment:
I don't think you want to do this. Anything that is not a PRO ID will never be exported. See https://www.tutorialspoint.com/python/python_loop_control.htm